### ISSUE TITLE
Poor initial galaxy fit

### ISSUE THREAD

**Issue by kbarbary** [snfactory/cubefit#25]
SNF20060609-002 has a poor initial galaxy fit in both channels. There is only one final ref (the last epoch/column) so the inital galaxy fit is also the final galaxy fit.

**B**
![snf20060609-002_b_timeseries](https://cloud.githubusercontent.com/assets/970135/11385152/d7a1450c-92ca-11e5-90f4-3e5320d7ad32.png)

**R**
![snf20060609-002_r_timeseries](https://cloud.githubusercontent.com/assets/970135/11385176/0d40b94a-92cb-11e5-8876-b9a64c2283a4.png)

The model is generally flatter than the data in both bands. It's possible that this is due to having a too-high spatial hyperparameter. Seems surprising though, as I haven't seen this in other SNe with similar-looking data.

**Comment by kbarbary** [comment]
I played around with the hyperparameters for this one. Oddly enough, the wavelength hyperparameter had a much larger effect. Reducing it from `mu_wave=0.07` to `mu_wave=0.001` gave the results shown below. Left is the underlying unconvolved model, right is the data, model, residual for the master ref epoch. Compare to the last column in the previous post.

**B**

![image](https://cloud.githubusercontent.com/assets/970135/11462763/25f3f32a-96cd-11e5-92f1-bc85cad9add3.png) ![image](https://cloud.githubusercontent.com/assets/970135/11462743/fc36e858-96cc-11e5-8c93-bdbce113df5b.png)

**R**

![image](https://cloud.githubusercontent.com/assets/970135/11462776/57a37d50-96cd-11e5-96ba-5abcb6d3cd1a.png) ![image](https://cloud.githubusercontent.com/assets/970135/11462779/6e59abaa-96cd-11e5-8eff-4ff474a9ea77.png)

There seems to be a clear Einstein ring, nicely pulled out of the data by cubefit. It's more evident in earlier epochs with better seeing, but these epochs are not used to constrain the galaxy model. Pretty cool.

**Comment by kbarbary** [comment]
I tried running my enitre test set (~30 SNe) with `mu_wave=0.001` and resulting residuals were almost always worse, or at best the same (with the exception of this one SN). Plus it usually took longer to fit.

My hypothesis is that this is because the lower hyperparameter allows overfitting to noise in the references.

Looking at slices in wavelength, the model seems to vary dramatically from one wavelength to another (for both `mu_wave=0.001` and `mu_wave=0.07`) so I'm a bit puzzled about why we're not able to get a better fit with the higher hyperparameter value for this one SN; if anything it seems like the hyperparameter is too low. In general, the hyperparameters could stand some more investigation.

**Comment by ycopin** [comment]
Could it be related to the presence of strong gaseous structures different of the continuum one? Could you make the Ha image? (and/or ask Mickael & Co. to do it systematically in his host studies)
